website/versioned_docs/version-0.2.0/getting-setup.md

```diff
@@ -0,0 +1,109 @@
+# Getting Setup With Bitcoin-S
+
+## Step 1: Java and Scala
+
+The first step in getting setup will be getting the [Java Development Kit](https://www.oracle.com/java/technologies/javase-downloads.html) (JDK) installed on your machine. Bitcoin-S works best with Java 8 but _should_ also work with Java 11.
+
+Once java is setup on your machine (try running `javac -version`), you are ready to download and install the [Scala Build Tool](https://www.scala-sbt.org/download.html) (sbt). Note that running `sbt` for the first time will take a while.
+
+## Step 2: Bitcoin-S Repository
+
+Now, it is time to clone the [Bitcoin-S repository](https://github.com/bitcoin-s/bitcoin-s/) by running
+
+```bashrc
+git clone git@github.com:bitcoin-s/bitcoin-s.git
+```
+
+or alternatively, if you do not have ssh setup with github, you can run
+
+```bashrc
+git clone https://github.com/bitcoin-s/bitcoin-s.git
+```
+
+Next, you will want to execute the commands
+
+```bashrc
+cd bitcoin-s
+git submodule update
+```
+
+to download the secp256k1 submodule.
+
+You should be able to test your secp256k1 installation by running `sbt console` in your bitcoin-s directory and then running
+
+```scala
+scala> import org.bitcoin._
+import org.bitcoin._
+
+scala> Secp256k1Context.isEnabled()
```

Comment:
Add a warning that this may cause an error message about logging but is not a concern